clang-tools-extra/docs/clang-tidy/checks/misc/coroutine-hostile-raii.rst

```diff
@@ -0,0 +1,49 @@
+.. title:: clang-tidy - misc-coroutine-hostile-raii
+
+misc-coroutine-hostile-raii
+====================
+
+This check detects hostile-RAII objects which should not persist across a 
```

Comment:
this should start with: "Detects when objects of certain hostile RAII types persists across suspension points in a coroutine.
  Such hostile types include scoped-lockable types and types belonging to a configurable denylist."